target/linux/generic/pending-5.10/130-add-linux-spidev-compatible-si3210.patch

```diff
@@ -1,18 +0,0 @@
-From: Giuseppe Lippolis <giu.lippolis@gmail.com>
```

Comment:
I wonder why this patch was removed? I see that the same one was removed from pending-5.15, but there was superseeded by two upstream patches.

Reply:
This patch is replaced by the two from this PR to ensure that the manufacturer name is matching the one already used upstream.
I did not add them for 5.10 as I have been told in [another discussion](https://github.com/openwrt/openwrt/pull/11602#discussion_r1056620030) that this is not relevant anymore. I can add them if needed, do you know if 5.10 is still being used on master?

Reply:
5.10 is not relevant for IPQ40xx that you were targeting, but if its in generic then 5.10 matters as there are targets still using it

Reply:
Ok, I was not aware that it was still being used. I will add the patch for 5.10 as well. Sorry for the confusion!

Reply:
No worries

Reply:
fixed